### ISSUE TITLE
[Feature Request] - Accessible social anchor elements inside Footer

### ISSUE THREAD

**Issue by 0xabdulkhalid** [amupedia2021/Project-Amupedia#262]
### Is your feature request related to a problem ? Please describe. ✍️

```bash
Enhancement:
- The social anchor elements on `footer` in accessible, so users who relay on screen readers may face issues to properly navigate to responsible destination
```

### Describe breifly how would you implement the same.

```bash
Solution_steps:
- We can resolve the issue with adding an `aria-label` attribute for those anchor elements
- The aria-label attribute is used to provide a text alternative for an element that is not visible to screen readers. This is important for accessibility, as it allows people who are blind or have low vision to understand the purpose of the element.
```

### Screenshot:
![Screenshot 2023-05-27 at 12-27-48 Amupedia](https://github.com/amupedia2021/Project-Amupedia/assets/92252895/896925f9-9e70-47c9-9122-1e65b8399921)
From
```html

<a href="https://www.instagram.com/amu.pedia/?hl=am-et" target="_blank" rel="noreferrer">
 <svg>
</a>
````
To
```html

<a href="https://www.instagram.com/amu.pedia/?hl=am-et"  aria-label="Visit us on Instagram"  target="_blank" rel="noreferrer">
 <svg>
</a>
````
 This snippet `aria-label="Visit us on Instagram"` is going to be added which improves accessibility of this site

<br>

### Additional context

Additional context

### Are you contributing for a program?

GSSoC2023

### Code of Conduct

- [X] I have read and agree to the [CONTRIBUTING GUIDELINES](https://github.com/amupedia2021/Project-Amupedia/blob/master/CONTRIBUTING.md) of this project.

**Comment by AmanJain18** [comment]
Please add a screenshot of what you will implement for better understanding... Re edit Issue...

**Comment by 0xabdulkhalid** [comment]
> Please add a screenshot of what you will implement for better understanding... Re edit Issue...

@AmanJain18, I updated the issue. Currently we can't visually show the changes in screenshot because it's for disabled peoples who relay on screen readers.

Added the snippet which i'm going to add to implement accessiblity you can refer it.

**Comment by 0xabdulkhalid** [comment]
Pull request has been opened, waiting for your review @AmanJain18 👀

**Comment by AmanJain18** [comment]
> Pull request has been opened, waiting for your review @AmanJain18 👀 

I have checked and suggested changes...
